src-pwa/register-service-worker.js

```diff
@@ -19,7 +18,12 @@ register(process.env.SERVICE_WORKER_FILE, {
   },
   updated(registration) {
     // registration -> a ServiceWorkerRegistration instance
-    console.log('New content is available; please refresh.')
+    // console.log(Dialog)
+    var popup = confirm('New content is available please refresh!')
+    if (popup === true) {
+      registration.waiting.postMessage('skipWaiting')
+      window.location.reload(true)
```

Comment:
window.location.reload(true) should reload be called here ?
coz the delete cache is async, what if reload happens when cache not yet clear ?